src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: 726px;
+  padding: 10px;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  
+    @include square($day-size);
+
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
```

Comment:
it's better to put into reset.scss or normalize.scss

Reply:
ok